go.mod

```diff
@@ -68,3 +68,5 @@ require (
 )
 
 replace k8s.io/klog => github.com/simonpasquier/klog-gokit v0.1.0
+
+go 1.13
```

Comment:
are you ok if we wait for 1.13 to be released first?

Reply:
My bad, I should drop this.

Reply:
Dropped this. My bad.